opennms-webapp/src/main/webapp/includes/grafana-box.jsp

```diff
@@ -117,9 +125,14 @@
                     }
                 }
                 if (showDashboard) {
-                    $('#dashboardlist').append('<a href="<%=grafanaProtocol%>://<%=grafanaHostname%>:<%=grafanaPort%>/dashboard/' + val['uri'] + '"><span class="glyphicon glyphicon-signal" aria-hidden="true"></span>&nbsp;' + val['title'] + "</a><br/>");
+                    if (limit < 1 || count++ < limit) {
+                        $('#dashboardlist').append('<a href="<%=grafanaProtocol%>://<%=grafanaHostname%>:<%=grafanaPort%>/dashboard/' + val['uri'] + '"><span class="glyphicon glyphicon-signal" aria-hidden="true"></span>&nbsp;' + val['title'] + "</a><br/>");
+                    }
                 }
             };
+            if (limit > 0 && count > limit) {
+                $('#dashboardlist').append('<a href="graph/grafana.jsp">&nbsp;&nbsp;View List of All Dashboards</a><br/>');
```

Comment:
What about to also use a icon instead of using &nbsp;&nbsp;
Maybe "glyphicon glyphicon-th-list" or. something like this...